### PR TITLE
Cpp limits

### PR DESCRIPTION
**Pull request by molpopgen**
I'm working on wrapping a C++ library where numeric_limits is needed.  This .pxd works for my purposes.

One caveat is that Cython doesn't appear to support static member variables, so I've declared them non-static.  I've documented that a user must then create an object of that type in order to access those values.   I'm guessing such access are edge cases, but that having the static function is useful.

### CODE REVIEW

**Comment by robertwb** [comment]
Correct, there's currently no way to declare static members. A test would be nice.

**Comment by molpopgen** [comment]
I can work up a test, but I'm not sure where to put it. In looking at the
repo, I'm having trouble locating tests for the libcpp stuff--any pointers
on where to look?

**Comment by robertwb** [comment]
How about putting a couple tests in https://github.com/cython/cython/blob/master/tests/run/libcpp_all.pyx?

**Comment by molpopgen** [comment]
I've generated some very basic tests, which seem to be passing.

**Comment by robertwb** [comment]
Thanks!

File: Cython/Includes/libcpp/limits.pxd [hunk 1]
```diff
@@ -0,0 +1,61 @@
+cdef extern from "limits" namespace "std" nogil:
+   enum float_round_style:
+        round_indeterminate       = -1
```
Comment (robertwb) [inline Cython/Includes/libcpp/limits.pxd]: These literal values are not needed; just omit them and we'll use the underlying C++ ones.
